recipes/xcube/meta.yaml

```diff
@@ -0,0 +1,87 @@
+{% set name = "xcube" %}
+{% set version = "0.2.0.dev1" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/dcs4cop/{{ name }}/archive/{{ version }}.tar.gz"
+  sha256: "2617a04fb38511b9bba629e57c5358c4c2cb2384ec216334dd6f6947b43a80a3"
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    # Put any entry points (scripts to be generated automatically) here.
+    - xcube = xcube.cli.cli:main
+    - xcube-gen = xcube.cli.gen:main
+    - xcube-restime = xcube.cli.restime:main
+
+requirements:
+  build:
+    # Python
+    - python =3.7
+    - pip =19.2
+  run:
+    # Python
+    - python =3.7
+    # Required
+    - affine =2.2
+    - blas =*=mkl
+    - click =7.0
+    - cmocean =2.0
+    - dask =1.2
+    - fiona =1.8
+    - gdal =2.4
+    - matplotlib =3.0
+    - netcdf4 =1.5.0
+    - numba =0.43
+    - numpy =1.16
+    - pandas =0.24
+    - pillow =6.0
+    - pip =19.2
+    - proj4 =5.2
+    - pyyaml =5.1
+    - rasterio =1.0
+    - s3fs =0.2
+    - scipy =1.2
+    - setuptools =41.0
+    - shapely =1.6
+    - tornado =6.0
+    - xarray =0.12.3
+    - zarr =2.3.2
+    - sphinx =2.0.1
+test:
+  # Python imports
+  imports:
+    - xcube
+    - xcube.api
+    - xcube.cli
+    - xcube.util
+
+  commands:
+    # You can put test commands to be run here.  Use this to test that the
+    # entry points work.
+    - xcube --help
+
+  requires:
```

Comment:
`pytest` is not used in the tests, please remove

Reply:
Removed.